src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -87,12 +93,61 @@ public static IHostBuilder AddScriptHost(this IHostBuilder builder, ScriptApplic
             });
 
             // WebJobs configuration
-            return builder.AddScriptHostCore(applicationOptions, metricsLogger, configureWebJobs, loggerFactory);
+            builder.AddScriptHostCore(applicationOptions, configureWebJobs, loggerFactory);
+
+            // Allow FunctionsStartup to add configuration after all other configuration is registered.
```

Comment:
>// Allow FunctionsStartup to add configuration after all other configuration is registered. [](start = 12, length = 91)

Doing it here would also mean configurations are re-registered with every jobhost restart, right? If some of can be file operations, could it mean potential perf impact upon every jobhost restart?

Reply:
Yes, but it's the same as any other Startup.cs class -- we need to re-run them whenever something changes as it can have code changes in them.